### ISSUE TITLE
melpa-stable not working, seems like MELPA changed URI ?

### ISSUE THREAD

**Issue by joakim-tjernlund** [jauhien/gs-elpa#12]

**Comment by CaptainBloodz** [comment]
same with melpa

**Comment by CaptainBloodz** [comment]
[https://bugs.gentoo.org/show_bug.cgi?id=734076](url)
This gentoo bug report page contains a patch for both URI update.

**Comment by dol-sen** [comment]
Since Jauhien has not been active, I have started gathering some changes from the forks of this repo and going to make a new release.  If you have any patches/pull-requests please make them there.

https://github.com/dol-sen/gs-elpa

**Comment by haschka** [comment]
I can confirm this issue ! As a quick fix you can edit : 
/etc/g-sorcery/gs-elpa.json and fix the urls there. 

they basically moved away from milkbox. the new addresses are: 

http://melpa.org/packages for melpa
and
http://stable.melpa.org/packages for melpa-stable

**Comment by dol-sen** [comment]
yes, I have applied that in my fork already.  I have a little more to do before I make a new release, change the SRC_URI in the live ebuild.

**Comment by ulm** [comment]
Fixed in gs-elpa-0.2 (hosted on Gentoo infra: https://gitweb.gentoo.org/proj/gs-elpa.git).

**Comment by CaptainBloodz** [comment]
Fixed

**Comment by ulm** [comment]
Unfortunately, I don't have the rights to close this.
